tracing/zipkin/span_test.go

```diff
@@ -0,0 +1,45 @@
+package zipkin_test
+
+import (
+	"testing"
+
+	"github.com/go-kit/kit/tracing/zipkin"
+)
+
+func TestAnnotateBinaryEncodesKeyValueAsBytes(t *testing.T) {
+	key := "awesome-bytes-test"
+	value := []byte("this is neat")
+
+	span := &zipkin.Span{}
+	span.AnnotateBinary(key, value)
+
+	encodedSpan := span.Encode()
+	annotations := encodedSpan.GetBinaryAnnotations()
+
+	if annotations[0].Key != key {
+		t.Errorf("Error: expected %s got %s", key, annotations[0].Key)
+	}
+
+	if string(annotations[0].Value) != string(value) {
+		t.Errorf("Error: expected %s got %s", string(value), string(annotations[0].Value))
+	}
```

Comment:
Here, and in other test cases, please use the mostly-standard form leveraging want and have vars. And when comparing bytes, I think bytes.Compare is nicer than string conversions.

```go
if want, have := value, annotations[0].Value; bytes.Compare(want, have) != 0 {
    t.Errorf("want %s, have %s", want, have)
}
```